p2p/server/server.go

```diff
@@ -316,7 +317,7 @@ func (s *Server) queueHandler(ctx context.Context, stream network.Stream) bool {
 }
 
 // Request sends a binary request to the peer.
-func (s *Server) Request(ctx context.Context, pid peer.ID, req []byte) ([]byte, error) {
+func (s *Server) Request(ctx context.Context, pid peer.ID, req []byte, extraProtocols ...string) ([]byte, error) {
```

Comment:
with extraProtocols you fallback to hs/1 ?

Reply:
yes